### ISSUE TITLE
unnecessary-else throws error

### ISSUE THREAD

**Issue by felixfbecker** [palantir/tslint#4622]
### Bug Report

- __TSLint version__: 5.15.0
- __TypeScript version__: 3.4.1
- __Running TSLint via__: (pick one) CLI 

#### TypeScript code being linted

https://github.com/sourcegraph/sourcegraph/blob/09e9e182d707526f1d4438e9e80b736a4f0c8dcf/web/src/components/FilteredConnection.tsx#L514

with `tslint.json` configuration:

```json
    "unnecessary-else": true,
```

#### Actual behavior
```
The 'unnecessary-else' rule threw an error in '/Users/felix/git/sourcegraph/web/src/components/FilteredConnection.tsx':
TypeError: Cannot read property 'kind' of undefined
    at getJumpStatement (/Users/felix/git/sourcegraph/node_modules/tslint/lib/rules/unnecessaryElseRule.js:70:18)
    at cb (/Users/felix/git/sourcegraph/node_modules/tslint/lib/rules/unnecessaryElseRule.js:56:19)
    at visitNode (/Users/felix/git/sourcegraph/node_modules/typescript/lib/typescript.js:16505:24)
    at Object.forEachChild (/Users/felix/git/sourcegraph/node_modules/typescript/lib/typescript.js:16755:21)
    at cb (/Users/felix/git/sourcegraph/node_modules/tslint/lib/rules/unnecessaryElseRule.js:63:19)
    at visitNodes (/Users/felix/git/sourcegraph/node_modules/typescript/lib/typescript.js:16514:30)
    at Object.forEachChild (/Users/felix/git/sourcegraph/node_modules/typescript/lib/typescript.js:16740:24)
    at cb (/Users/felix/git/sourcegraph/node_modules/tslint/lib/rules/unnecessaryElseRule.js:63:19)
    at visitNode (/Users/felix/git/sourcegraph/node_modules/typescript/lib/typescript.js:16505:24)
    at Object.forEachChild (/Users/felix/git/sourcegraph/node_modules/typescript/lib/typescript.js:16754:21)
```
#### Expected behavior
No error

**Comment by adidahiya** [comment]
just released 5.15.0, try that out, it has some bugfixes for this rule

**Comment by felixfbecker** [comment]
Ah sorry, caching I guess
